FuelSDK-CSharp/ETClient.cs

```diff
@@ -91,8 +94,12 @@ public ETClient(NameValueCollection parameters = null, RefreshState refreshState
                 {
                     configSection.RestEndPoint = parameters["restEndPoint"];
                 }
+                if (parameters.AllKeys.Contains("oAuth2AuthenticationMode"))
```

Comment:
I think the config attribute should be named `useOAuth2Authentication` since it represents a boolean value.

Reply:
Done.